SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -42,13 +42,13 @@ USpatialGDKEditorSettings::USpatialGDKEditorSettings(const FObjectInitializer& O
 	, bShowSpatialServiceButton(false)
 	, bDeleteDynamicEntities(true)
 	, bGenerateDefaultLaunchConfig(true)
-	, RuntimeVariant(ESpatialOSRuntimeVariant::Standard)
 	, StandardRuntimeVersion(SpatialGDKServicesConstants::SpatialOSRuntimePinnedStandardVersion)
-	, CompatibilityModeRuntimeVersion(SpatialGDKServicesConstants::SpatialOSRuntimePinnedCompatbilityModeVersion)
+	, bUseGDKPinnedInspectorVersion(true)
+	, InspectorVersionOverride(TEXT(""))
 	, ExposedRuntimeIP(TEXT(""))
 	, bAutoStartLocalDeployment(true)
 	, bSpatialDebuggerEditorEnabled(false)
-	, AutoStopLocalDeployment(EAutoStopLocalDeploymentMode::OnExitEditor)
+	, AutoStopLocalDeployment(EAutoStopLocalDeploymentMode::OnEndPIE)
```

Comment:
Cool, is this because starting deployments should be faster?

Reply:
Yup, the default behaviour should now be to always restart. We want to leave the ability to keep it up for other debugging purposes though, hence just changing the default.